scss/_transitions.scss

```diff
@@ -14,16 +14,21 @@
   &.show {
     display: block;
   }
+  &.flexshow {
+    display: flex;
+  }
 }
 
 tr {
-  &.collapse.show {
+  &.collapse.show,
+  &.collapse.flexshow {
```

Comment:
Should `.flexshow` be named `.flex-show` to be consistent with the hyphen separation used by the other `.flex-*` classes?